### ISSUE TITLE
Review accessibility report for Helix

### ISSUE THREAD

**Issue by riarenas** [dotnet/dnceng#4010]
Review the detailed report and file issues for all work.

### Release Note Category
- [ ] Feature changes/additions 
- [X] Bug fixes
- [ ] Internal Infrastructure Improvements
### Release Note Description
Added styling to links in the footer of the Helix website.

**Comment by riarenas** [comment]
The detailed report is available [here](https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=8512cf21-a1bd-4b5d-9689-08d8390dcd48) (requires auth)

**Comment by riarenas** [comment]
Removing my assignment for now due to shifting priority

**Comment by riarenas** [comment]
All the findings are related to the [footer](https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service?path=/src/ServiceFabric/Helix/HelixAPI/Pages/Shared/_Footer.cshtml) in https://helix.dot.net

> Fix ONE of the following:
The link has insufficient color contrast of 1.11:1 with the surrounding text. (Minimum contrast is 3:1, link text: #086ad4, surrounding text: #6c757d)
The link has no styling (such as underline) to distinguish it from the surrounding text

Seems like adding underline to the links might fix them all

**Comment by riarenas** [comment]
PR: https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service/pullrequest/42742

**Comment by riarenas** [comment]
This has been deployed to production. The next scan is set up to run on 10/1 and I will wait to see the results of that scan before closing or sending this back for follow up work.

**Comment by riarenas** [comment]
The https://webcompliance.trafficmanager.net/ComplianceResults/8512cf21-a1bd-4b5d-9689-08d8390dcd48/Helix website still shows the accessibility scan as failed, but there are no detailed results for me to examine. Dropping this back to the backlog for investigation on what needs to happen to make the report green.

**Comment by missymessa** [comment]
Going to put this back into the backlog. Website says the next accessibility scan is: 12/3/2024, so we can check back in a month to ensure we're compliant.

**Comment by riarenas** [comment]
I made the changes by the end of September, and I think there have been multiple scans with the changes in place that have resulted in the same error reporting. I think it would be good to see if the changes were not sufficient or what's going on with the scanner?

**Comment by riarenas** [comment]
@ilyas1974 reached out to the accessibility scanner team and they recommended a fix: https://dnceng.visualstudio.com/internal/_git/dotnet-helix-service/pullrequest/45159
